### PR TITLE
Fix missing footer, so auto-refresh and fullscreen are unavailable

### PR DESCRIPTION
**Pull request by flamber**
Fixes https://github.com/metabase/metabase-enterprise/issues/456, but adding here, since it also affects Premium Embedding.

### CODE REVIEW

**Review by robdaemon — APPROVED** [review]

File: frontend/src/metabase/public/components/EmbedFrame.jsx [hunk 1]
```diff
@@ -61,9 +61,7 @@ export default class EmbedFrame extends Component {
     } = this.props;
     const { innerScroll } = this.state;
 
-    const showFooter =
-      !MetabaseSettings.hideEmbedBranding() ||
-      (actionButtons && actionButtons.length > 0);
+    const showFooter = !MetabaseSettings.hideEmbedBranding() || actionButtons;
```
Comment (salsakran) [inline frontend/src/metabase/public/components/EmbedFrame.jsx:64]: @robdaemon does this bring back the "powered by Metabase" section?

Comment (robdaemon) [reply frontend/src/metabase/public/components/EmbedFrame.jsx:64]: Not that I know of, this condition is effectively the same?

Comment (salsakran) [reply frontend/src/metabase/public/components/EmbedFrame.jsx:64]: Can you please make sure? I believe the footer contains "powered by Metabase", and that's why we hide it. If `hideEmbedBranding` is true, we must not show that attribution link

Comment (robdaemon) [reply frontend/src/metabase/public/components/EmbedFrame.jsx:64]: Yep, the desired behavior is still maintained with this change: https://github.com/metabase/metabase/blob/master/frontend/src/metabase/public/components/EmbedFrame.jsx#L116-L123

Comment (salsakran) [reply frontend/src/metabase/public/components/EmbedFrame.jsx:64]: cool, thanks for checking!

**Pull request merged**